backend/models.py

```diff
@@ -178,4 +206,14 @@ class CapstoneInfo(db.Model):
     users = relationship(User)
 
     def __repr__(self):
-        return '<Capstone info {}{}>'.format(self.title)
\ No newline at end of file
+        return '<Capstone info {}>'.format(self.title)
+
+# Generate marshmallow Schemas from the model
+class ProjectSchema(ma.Schema):
+    class Meta:
+        # Fields to expose
+        fields = ("id","user_id", "title", "abstract", "keywords", "feature",
+                "los", "custom_los", "hsr_review", "last_updated")
```

Comment:
Will ```last_updated``` be provided by users? I think it is better to have the backend generate it for us?